content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -167,46 +202,25 @@ metadata:
 spec:
   hosts:
   # must be of form name.namespace.global
-  - bar.ns2.global
+  - httpbin.bar.global
   location: MESH_INTERNAL
   ports:
   - name: http1
-    number: 8080
+    number: 8000
     protocol: http
-  - name: tcp2
-    number: 9999
-    protocol: tcp
   resolution: DNS
   addresses:
   - 127.255.0.2
   endpoints:
-  - address: <IPofCluster2IngressGateway>
+  - address: ${GATEWAY_IP_CLUSTER1}
     network: external
     ports:
       http1: 15443 # Do not change this port value
-      tcp2: 15443 # Do not change this port value
   - address: istio-egressgateway.istio-system.svc.cluster.local
```

Comment:
@rshriram  Here's the thing I don't quite follow. We have two endpoints, one for remote cluster's ingress, one for local cluster's egress. Wouldn't Pilot generate one Endpoint per entry here? How can Pilot enforce the traffic all go thru local egress?

Reply:
Because only egress gateawy will see the remote cluster's endpoint. In the helm charts, we set the REQUESTED_NETWORK_VIEW for the egress gateway to external.. (and in the service entry above, we provide the remote cluster's ingress gateway endpoint as an "external" network endpoint in the service entry).

So yes, pilot will generate 2 endpoints, but one for sidecar and a different one for egress gateway

Reply:
its a metadata. ISTIO_META_REQUESTED_NETWORK_VIEW